clients/drcachesim/common/trace_entry.h

```diff
@@ -726,6 +740,7 @@ typedef enum {
      * 1 (success).
      */
     OFFLINE_FILE_TYPE_BLOCKING_SYSCALLS = 0x800,
+    OFFLINE_FILE_TYPE_SYSCALL_PT = 0x1000, /**< Syscalls' PT tracings are included. */
```

Comment:
Explain what "PT" means.  Also, if we use equivalent technology on other platforms, we could use this same filetype (the arch should be enough to identify the technology), right?  So rename to OFFLINE_FILE_TYPE_KERNEL_SYSCALLS or something?

Reply:
Done.